modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -12,6 +12,15 @@ WARNING: `CHE_INFRA_KUBERNETES_NAMESPACE` and `CHE_INFRA_OPENSHIFT_PROJECT` are
 
 WARNING: By default, only one workspace in the same {orch-namespace} can be running at one time. See xref:running-more-than-one-workspace-at-a-time.adoc[].
 
+[WARNING]
+====
+By {kubernetes} definition, the maximum length of namespace name is limited to 63 characters, including the evaluated placeholders.
```

Comment:
The name also needs to be a valid DNS name.
Also, I would maybe rephrase this to slightly simplify the language (IMHO):

{kubernetes} limits the length of a namespace name to 63 characters (this includes the evaluated placeholders). Additionally, the names (after placeholder evaluation) must be valid DNS names.

Reply:
much better, updated